### DISCUSSION TITLE
boostrapper doesnt work

### DISCUSSION THREAD

**Discussion by wdhfgueyhfeioruferrf** [quivings/Solara#168] [Q&A]
when i run it it does the normal stuff like adding the exploit files but then it just disappears and solara tab doesnt show up
and yes i have real time protection turned off 

https://github.com/user-attachments/assets/b54a8cc2-f39e-4423-afd9-768fdb88c300

**Comment by chicken-nugget1104** [comment]
download solara.dir from here :3
